lib/Dialects/LinalgTransform/IR/LinalgTransformOps.cpp

```diff
@@ -886,5 +887,21 @@ transform::TileToLinalgExtTileOp::applyToOne(TilingInterface target) {
   return functional::applyAt(target, tileSeq);
 }
 
+FailureOr<scf::ForOp> transform::RewriteLinalgExtTileToScfForOp::applyToOne(
+    linalg_ext::TileOp target) {
+  linalg_ext::TileOpToSCFRewriter pattern(this->getContext());
+  auto functionalRewrite =
+      [&](linalg_ext::TileOp op,
+          PatternRewriter &rewriter) -> FailureOr<scf::ForOp> {
+    auto result = pattern.returningMatchAndRewrite(op, rewriter);
+    if (failed(result))
+      return failure();
+    return result;
+  };
+  auto rewriteSeq =
+      functional::SequenceBuilder().begin(std::move(functionalRewrite));
+  return functional::applyAt(target, rewriteSeq);
```

Comment:
return functional::applyAt(target, functionalRewrite); should be sufficient here since the sequence contains only one rewrite.